server/public/model/emoji.go

```diff
@@ -14,6 +14,8 @@ const (
 	EmojiSortByName    = "name"
 )
 
+var GiphySdkKey string
```

Comment:
nit: do we move this to the `config.go` where it's referenced? Seems we don't use it here in this emoji file

Reply:
this is the LDFlag linker variable

Reply:
Yes, but Go works at a package level. Not at a file level. We are specifying `model.GiphySdkKey=<>` in the Makefile, so that means this variable can be specified in any file in the model package. So Maria is right, it's better to move this to the config.go file where it's used. Variables are better declared close to where they are used.

Reply:
yes moved to config.go